web/src/test/kotlin/no/nav/su/se/bakover/web/TestEnvironment.kt

```diff
@@ -87,3 +87,16 @@ fun TestApplicationEngine.defaultRequest(
         setup()
     }
 }
+
+fun TestApplicationEngine.authorizedRequest(
```

Comment:
Kanskje ikke nødvendig å lage denne type extensions for å få gjort såpass lite? Trenger vel fort like mye kode for å ta denne i bruk som en vanlig handleRequest-block?  Kan kanskje virke litt forvirrende i forhold til "defaultRequest" når den har dette navnet.